src/main/java/org/alliancegenome/curation_api/interfaces/crud/AlleleCrudInterface.java

```diff
@@ -38,7 +41,7 @@ public interface AlleleCrudInterface extends BaseCurieCrudInterface<Allele>, Bas
 
 	@Override
 	@GET
-	@JsonView(View.FieldsAndLists.class)
+	@JsonView(View.Allele.class)
```

Comment:
I think it would be better to rename this to View.AllleleView I know its redundant but at least we won't have two classes in the project named "Allele".

Reply:
Done